### PR TITLE
Prometheus Monitoring

### PR DESCRIPTION
**Pull request by ralgozino**
Hello team!

This PR adds the metrics port to the service definition and the service monitor definition for prometheus scraping in order to be able to see Gatekeeper metrics on a Grafana Dashboard

### CODE REVIEW

File: docs/releases/unreleased.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Release notes
+
+## Changelog
+
+Changes between `1.0.2` and this release: `TBD`
```
Comment (angelbarrera92) [inline docs/releases/unreleased.md:5]: Love it!

**Comment by angelbarrera92** [comment]
Now that we are including a SeriviceMonitor.... should we document that this module depends of having the fury-kubernetes-monitoring module previously available in the cluster?

**Comment by ralgozino** [comment]
> Now that we are including a SeriviceMonitor.... should we document that this module depends of having the fury-kubernetes-monitoring module previously available in the cluster?

That's a very valid point. Let me fix that.

**Comment by ralgozino** [comment]
@angelbarrera92  I've added to the readme (and the changelog) the new requirement, thanks for spotting that! :)

**Pull request merged**